### ISSUE TITLE
ODK formatting & image bases questions handling

### ISSUE THREAD

**Issue by choxx** [Samagra-Development/quml2xform#5]
> Also Image questions need to be handled separately. Same is the case with formatting. ODK allows markdown based formatting. So html to makrdown conversion needs to happen as well.
>https://github.com/Sunbird-Ed/program-service/blob/8dfbf3efb65e886933bb0d788dd5281000d1aa07/src/service/print/pdf.js#L209
https://github.com/Sunbird-Ed/program-service/blob/8dfbf3efb65e886933bb0d788dd5281000d1aa07/src/service/print/pdf.js#L274
This method can be directly used.

I tweaked the first part and am now excluding **<strong>** tags from stripping out: [ref](https://github.com/choxx/quml2xform/blob/78f277a8076fea7354f9b8a7d9e7989a51373c8e/src/quml-to-odk/quml-to-odk.service.ts#L398)

For the second, I'm not sure where exactly & how to use this [function](https://github.com/Sunbird-Ed/program-service/blob/8dfbf3efb65e886933bb0d788dd5281000d1aa07/src/service/print/pdf.js#L274)

---

Please also elaborate how to handle media-based questions.

**Comment by ChakshuGautam** [comment]
@choxx is this closed now?

**Comment by choxx** [comment]
> @choxx is this closed now?

Yes, except for the inline HTML support in Android (which is to be clarified with Android dev separately as discussed)
